build.gradle.kts

```diff
@@ -1,3 +1,10 @@
+configurations.all {
```

Comment:
No idea what's going on here, but using latest SNAPSHOT cause issues

See (https://rewriteoss.slack.com/archives/C01A843MWG5/p1730980508003589)

Reply:
Thanks for pointing these out; fixed in
- https://github.com/openrewrite/rewrite-jenkins/pull/78